module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,21 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+struct StrCompare {
+    bool operator()(const std::string& str1, const std::string& str2) {
+        if (str1.size() == str2.size()) {
+            return str1 < str2;
+        }
+        return str1.size() < str2.size();
+    }
+};
```

Comment:
I will accept it ofc :) But try for yourself create lambda. When you will work as C++11 programmer you will always write lambda instead of struct.